create/create.go

```diff
@@ -12,10 +12,17 @@ func ReadTemplateFile(path string) ([]byte, error) {
 }
 
 func NewExtensionProject(extension core.Extension) error {
-	setup := NewProcess(
+	tasks := []Task{
 		MakeDir(extension.Development.RootDir),
 		CreateProject(extension),
-		InstallDependencies(extension.Development.RootDir),
+	}
+
+	if extension.Development.InstallDependencies == nil || *extension.Development.InstallDependencies {
```

Comment:
> Should we just make it a boolean and then update the test data to make the old extensions have this prop set to true?

Wouldn't this be a breaking change? I ended up going the pointer path because I couldn't find a way to give that attribute a default value. A blog post suggested to use pointers to represent the absence of the value. Open to suggestions.

Reply:
No one is expected to be using the old template so maybe we should just wait for the clean up PR to remove all the old templates and then remove the code to install dependencies?

Reply:
Also, CLI 2.0 is using a locked down version of the Go binary so we should be good to ship breaking changes